stable/elasticsearch/templates/tests/test-configmap.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: v1
```

Comment:
I wonder if there is any way to deploy this it only when doing test...

Reply:
this is how the other tests do it...

https://github.com/helm/charts/blob/bf877829d92dcb4bfc27163d60fcd403636915cc/stable/mysql/templates/tests/test-configmap.yaml#L1

https://github.com/helm/charts/blob/90e4ba91b3b0d933250bd3f5712a7199abb8703a/stable/consul/templates/test-config.yaml#L1